codecov.yml

```diff
@@ -0,0 +1,6 @@
+comment: off
+coverage:
+  status:
+    project:
+      default:
+        target: auto
```

Comment:
OK it looks like we are all in agreement that for now we should iterate on the target/threshold decisions, and set them to something that will not fail the context (e.g. 85% and 10% coverage drop). 
This way we will not impact developers by accidentally setting them too low (especially given the open latching issue), and we can start trending data at codecov.